src/components/Banner/Banner.tsx

```diff
@@ -55,202 +59,214 @@ export interface BannerProps {
   stopAnnouncements?: boolean;
 }
 
-export class Banner extends PureComponent<BannerProps, State> {
-  static contextType = FeaturesContext;
-  context!: React.ContextType<typeof FeaturesContext>;
-
-  state: State = {
-    showFocus: false,
-  };
+interface UseBannerPropsForStatusResult {
+  iconColor: IconProps['color'];
+  defaultIcon: IconProps['source'];
+  ariaRoleType?: 'alert';
```

Comment:
wonder if this should default to `status` 🤔

Reply:
Damn; Totally missed that. 😓  Reviewed the component without the diff and realized it was missing a default haha. Thank you!

Reply:
Fixed it. Assertions added in 5548b91fbb00c974d7dd3f5e0a8ed8cc72610848.